mx.mx/suite.py

```diff
@@ -519,6 +519,12 @@
             "optional" : True
           }
         },
+        "linux-musl" : {
+          "amd64" : {
+            "urls" : ["https://download.bell-sw.com/graalvm/ninja-1.10.1-x86_64-linux-musl.zip"],
```

Comment:
We would need to rebuild/repackage this on our own infrastructure. Is there anything special about this? Could you share your build instructions?

Reply:
I believe I've just cloned their repo and run `./configure.py --bootstrap`